src/components/slider/slider.e2e.ts

```diff
@@ -673,4 +673,49 @@ describe("calcite-slider", () => {
   it("is form-associated", () => formAssociated("calcite-slider", { testValue: 5 }));
 
   it("is form-associated with range", () => formAssociated("calcite-slider", { testValue: [5, 10] }));
+
+  it("groupSeparator: displays numbers as comma separated when prop is true and does not when false", async () => {
+    const page = await newE2EPage({
+      html: `<calcite-slider 
+                min="1000" 
+                max="10000" 
+                min-value="2500" 
+                max-value="5000" 
+                step="1000" 
+                ticks="1000" 
+                label-handles 
+                label-ticks 
+                style="width:${sliderWidthFor1To1PixelValueTrack}">
+            </calcite-slider>`
+    });
+    await page.waitForChanges();
+    const element = await page.find("calcite-slider");
+
+    const displayedValuesArray = async (): Promise<string[]> => {
+      await page.waitForChanges();
```

Comment:
I think you can get rid of this since you already wait for changes before calling this method.